File: segundo_cuatrimestre/java script/practica2/ej3.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function par (num) {
    if (num % 2 === 0) {
        console.log ("es par")
    } else {
        console.log ("es impar")
    }
}

rl.question ("Ingrese un numero", (num) => {
    num = Number(num)
    console.log ("el numero", par(num))
    rl.close()
})

File: facultad/programacion-1/semana12/ej3.c
#include <stdio.h>
#include <stdlib.h>

int espar (int num) {
    return (num == 0 || (num % 2 == 0));
}

int main() {
    int num;

    printf("Ingrese un numero: ");
    scanf("%d", &num);

    if (espar(num)) {
        printf("El numero %d es par.\n", num);
    } else {
        printf("El numero %d es impar.\n", num);
    }

    return 0;
}